### PR TITLE
Bumping up dependencies

### PR DESCRIPTION
**Pull request by dawid-nowak**
Bumping up dependencies to ensure that it works with Raspberry PI and Pimoroni Enviro mini board

### CODE REVIEW

**Comment by elpiel** [comment]
Thank you for the contribution.
I can release a new minor version as soon as the checks pass.

If you have example of how you use the crate I would love to see it.

File: Cargo.toml [hunk 1]
```diff
@@ -39,9 +39,9 @@ transform = ["embedded-graphics"]
...
 
 [dependencies]
-embedded-graphics-core = "0.3"
+embedded-graphics-core = "0.4"
 
-embedded-graphics = {version = "0.7", optional = true}
+embedded-graphics = {version = "0.8", optional = true}
 
 [dev-dependencies]
 embedded-graphics-simulator = "0.3"
```
Comment (elpiel) [inline Cargo.toml]: - [x] You should also update the simulator to version `0.5` as examples are failing.

**Review by elpiel — CHANGES_REQUESTED** [review]

File: Cargo.toml (hunk 1 above)
Comment (elpiel) [inline Cargo.toml]:
```suggestion
embedded-graphics-simulator = "0.5"
```

**Comment by elpiel** [comment]
Thank you once again for your contribution!
I will fix the MSRV before releasing

**Comment by dawid-nowak** [comment]
Super, thank you. The project is here https://github.com/dawid-nowak/all_in_one_enviro_mini_rs. Your library greatly helped to make it much better looking :)

**Comment by elpiel** [comment]
Great to hear that and thank you for sharing your project.
I was keen to see how you're using the crate.

Btw is there are reason you manually go over the pixel and drawing them on the display here?
You could use the `canvas.draw(disp)` directly iirc

https://github.com/dawid-nowak/all_in_one_enviro_mini_rs/blob/bf87fdee6c7631dadbed096911100356e1fd7664/src/display.rs#L100-L102

**Comment by dawid-nowak** [comment]
Yeah, draw uses ```target.draw_iter(pixels_iter)``` which turned out to be much slower than ```disp.fill_contiguous(&bb, pixels_iter)```.

To clarify, the differences are in the implemetnation of st7735 display [driver](https://github.com/sajattack/st7735-lcd-rs/blob/master/src/lib.rs) where ```fill_contiguous``` provides additional buffering and basically sending all pixels in one go.

**Comment by elpiel** [comment]
Hmm... Ok, that's great to know, I can make some tests and benchmark the result to improve the speed. I opened  #13 for this.
Thanks!

**Comment by dawid-nowak** [comment]
I don't think it is the problem with canvas but more with a display driver which implements DrawTarget trait.  For example, this driver only implements ```draw_iter``` which only updates the [buffer]( https://github.com/wjakobczyk/st7920/blob/e0518c54af26c352ebf912a49099f00cf915d4d0/src/lib.rs#L460) and one still needs to flush the buffer to send the data to the [display](https://github.com/wjakobczyk/st7920/blob/e0518c54af26c352ebf912a49099f00cf915d4d0/examples/graphics.rs#L72). 
Happy to chat on discord ```dawid.nowak``` if you want to discuss further.

**Pull request merged**